Extensions/Electron/electrontools.js

```diff
@@ -0,0 +1,200 @@
+gdjs.evtTools.electron = {
```

Comment:
Any chance you could add `// @ts-check` for this file?
Don't care about typing of `gdjs.evtTools.electron.window`, just add some parameter typing: `@param {boolean} activate`.

It's not super super useful here but better than nothing and can help in the future spot obvious mistake like variable name typos. Feel free to `// @ts-ignore` any error that is not easily solvable.